src/pages/api/graphql/typeDefs.ts

```diff
@@ -0,0 +1,8 @@
+import { readFileSync } from "fs";
+import gql from "graphql-tag";
+
+export const typeDefs = gql(
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.